packages/use-intl/test/useIntl.test.tsx

```diff
@@ -272,7 +272,7 @@ describe('formatNumber', () => {
 
       const error: IntlError = onError.mock.calls[0][0];
       expect(error.message).toBe(
-        'FORMATTING_ERROR: Invalid currency codes : unknown'
+        'FORMATTING_ERROR: Invalid currency code : unknown'
```

Comment:
Hmm, locally this is indeed "codes" for me with node v14.19.0. Seems like on CI there's the same result. Which node version are you using where you're seeing "code"?

Reply:
I am using node v16.13.1, so you were correct about the node version, sorry for "fixing" the test, i will undo it :)

Reply:
No problem! An upgrade to node v16 would definitely be interesting, but I'm not sure yet if there are other CI-related things to update then so that can be done at a later point for sure.